mod/groups/classes/Elgg/Groups/PageOwner.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Elgg\Groups;
+
+/**
+ * Handle page owner related hooks
+ *
+ * @since 4.0
+ * @internal
+ */
+class PageOwner {
+
+	/**
+	 * Helper handler to correctly resolve page owners on group routes
+	 *
+	 * @param \Elgg\Hook $hook "page_owner", "system"
+	 *
+	 * @return int|void
+	 */
+	public static function detectPageOwner(\Elgg\Hook $hook) {
```

Comment:
this was reported in a different ticket to verify if still needed?

Reply:
yep... #13064